test/smoke/run-smoke-suite.sh

```diff
@@ -0,0 +1,218 @@
+#!/bin/bash
+
+if [[ -n "${DEBUG}" ]]; then
+  set -x
+fi
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+export KIND_CLUSTER_NAME=${KIND_CLUSTER_NAME:-ingress-smoke-test}
+export KUBECONFIG="${KUBECONFIG:-$HOME/.kube/kind-config-$KIND_CLUSTER_NAME}"
+
+# Variables required for pulling Docker image with pytest
+SMOKE_REGISTRY_NAME="${SMOKE_REGISTRY_NAME:-dkr.wallarm.com}"
+SMOKE_IMAGE_PULL_SECRET_NAME="pytest-registry-creds"
+
+SMOKE_IMAGE_NAME="${SMOKE_IMAGE_NAME:-dkr.wallarm.com/tests/smoke-tests}"
+SMOKE_IMAGE_TAG="${SMOKE_IMAGE_TAG:-latest}"
+
+# Allure related variables
+ALLURE_ENDPOINT="${ALLURE_ENDPOINT:-https://allure.wallarm.com}"
+ALLURE_PROJECT_ID=${ALLURE_PROJECT_ID:-10}
+ALLURE_RESULTS="${ALLURE_RESULTS:-/tests/_out/allure_report}"
+ALLURE_UPLOAD_REPORT="${ALLURE_UPLOAD_REPORT:-false}"
+ALLURE_GENERATE_REPORT="${ALLURE_GENERATE_REPORT:-false}"
+
+# Pytest related variables
+CLIENT_ID="${CLIENT_ID:-5}"
```

Comment:
We need change tests to US cloud

Reply:
like in this : https://github.com/wallarm/docker-wallarm-node/pull/85/files

Reply:
done